ebamk2:
  controller:
    state:
      type: joint_state_controller/JointStateController
      publish_rate: 50
    position:
      joint_1:
        type: position_controllers/JointPositionController
        joint: joint_1
        pid: {p: 10.0, i: 0.0, d: 1.0}
      joint_2:
        type: position_controllers/JointPositionController
        joint: joint_2
        pid: {p: 5.0, i: 2.0, d: 1.0}
      joint_3:
        type: position_controllers/JointPositionController
        joint: joint_3
        pid: {p: 5.0, i: 2.0, d: 1.0}
      joint_4:
        type: position_controllers/JointPositionController
        joint: joint_4
        pid: {p: 5.0, i: 2.0, d: 1.0}
